app/styles/_exhibit.scss

```diff
@@ -38,6 +38,11 @@
   padding: $exhibit-layout-component-margin;
 }
 
+.exhibit__view-all {
+  padding: .2em .5em;
+  background-color: #fff;
+}
```

Comment:
What about adding this? 

    border: 1px solid #c74f0a;
    border-radius: 3px;
    top: -.2em;
    position: relative;